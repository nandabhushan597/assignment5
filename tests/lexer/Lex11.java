this is a file with 1 2 3 0x12 0xcafe 0xCAFE 0XaBc
  067 021l 123_45L 1___345 0b11011 0B101 0b1_0_1_0
  numbers
