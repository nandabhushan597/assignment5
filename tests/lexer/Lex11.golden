Right [KeywordT ThisK
IdentifierT "is"
IdentifierT "a"
IdentifierT "file"
IdentifierT "with"
LiteralT (IntL 1)
LiteralT (IntL 2)
LiteralT (IntL 3)
LiteralT (IntL 18)
LiteralT (IntL 51966)
LiteralT (IntL 51966)
LiteralT (IntL 2748)
LiteralT (IntL 55)
LiteralT (LongL 17)
LiteralT (LongL 12345)
LiteralT (IntL 1345)
LiteralT (IntL 27)
LiteralT (IntL 5)
LiteralT (IntL 10)
IdentifierT "numbers"]